package.json

```diff
@@ -11,6 +11,7 @@
     "react-google-login": "^5.2.2",
     "react-router-dom": "^5.2.0",
     "react-scripts": "^4.0.2",
+    "sopra-fs21-group-02-dogs-api": "^1.0.0",
```

Comment:
This is great! But as it stands I don't understand where NPM can find this package to install/reference the API code? Do we need to put it in `/node_modules` manually or tell `npm install` where to find it?

Reply:
I have published it here https://www.npmjs.com/package/sopra-fs21-group-02-dogs-api 
If you have an NPM account, I can invite you as contributor. 
And no manual work is needed, just npm install